package.json

```diff
@@ -54,9 +54,10 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "^2.59.1",
+    "@hmcts/ccd-case-ui-toolkit": "^2.62.16-RDM-5582-prerelease",
```

Comment:
Do we want to rely on a prerelease version in production code, or can we use a stable version?

Reply:
That is the ticket.